src/main/java/models/Article.java

```diff
@@ -2,16 +2,21 @@
 
 public class Article {
 
-    private int authorId;
-    private int postId;
+    private String postId;
+    private String authorId;
     private String title;
     private String authorFirstName;
     private String authorLastName;
     private String miniText;
     private int peopleTalking;
 
 
-    public Article(int authorId, int postId, String title, String authorFirstName, String authorLastName, String miniText, int peopleTalking){
+    public Article() {
+        super();
+    }
+
+    public Article(String authorId, String postId, String title, String authorFirstName, String authorLastName, String miniText, int peopleTalking) {
```

Comment:
Because Arango requires an empty constructor to be implemented whenever you use the model in the query i think ..bos kan beytalla3 exception w deh 7aletha e3melha f ba2eet el models mt2rfnresh 😂